CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ if(EMSCRIPTEN)
     set(XEUS_CPP_USE_SHARED_XEUS_CPP OFF)
     set(XEUS_CPP_BUILD_TESTS OFF)
     # ENV (https://github.com/emscripten-core/emscripten/commit/6d9681ad04f60b41ef6345ab06c29bbc9eeb84e0)
-    set(EMSCRIPTEN_FEATURES "${EMSCRIPTEN_FEATURES} -s \"EXTRA_EXPORTED_RUNTIME_METHODS=[ENV']\"")
+    set(EMSCRIPTEN_FEATURES "${EMSCRIPTEN_FEATURES} -s \"EXPORTED_RUNTIME_METHODS=[ENV']\"")
```

Comment:
`EXTRA_EXPORTED_RUNTIME_METHODS` had been deprecated in favour of `EXPORTED_RUNTIME_METHODS` since `2.0.18`